tests/integ/kms_utils.py

```diff
@@ -12,19 +12,26 @@
 # language governing permissions and limitations under the License.
 from __future__ import absolute_import
 
+import json
+
 from botocore import exceptions
 
-KEY_ALIAS = "SageMakerIntegTestKmsKey"
+PRINCIPAL_TEMPLATE = '["{account_id}", "{role_arn}", "arn:aws:iam::{account_id}:role/{' \
+                     'sagemaker_role}"] '
```

Comment:
seems like an awkward way to break the line. maybe instead do it right before the role arn?

Reply:
Sure!